routingv8/request.go

```diff
@@ -501,7 +502,28 @@ func (t *SpanAttribute) String() string {
 		return string(SpanAttributeNames)
 	case SpanAttributeMaxSpeed:
 		return string(SpanAttributeMaxSpeed)
+	case SpanAttributeLength:
+		return string(SpanAttributeLength)
 	default:
 		return invalid
 	}
 }
+
+type RouteImportRequest struct {
+	TransportMode TransportMode
+	// Which attributes to return in the response.
+	// If not specified defaults to SummaryReturnAttribute.
+	Return []ReturnAttribute
+	// The time of departure.
+	// If not specified the current time is used.
+	// To not take time into account use DepartureTimeAny.
+	DepartureTime string
```

Comment:
nit: why not make this a time.Time and handle the marshaling to HERE maps format in the function itself? the caller might not know what format to use?

Reply:
@odsod I agree with your reasoning, however this part was copied from the existing `RoutesRequest`, so I'm simply following the existing style. But I can also fix it. Which solution would you prefer?
1. Fixing it in both `RoutesRequest` and `RouteImportRequest`. This will keep the request types consistent, but will be a breaking change for any users of the library.
2. Fix just the `RouteImportRequest`, which won't break anything existing, but will make `RoutesRequest` and `RouteImportRequest` inconsistent.

Reply:
Ahh thanks for the heads up - I'd say we don't need to block this PR on that then - if we make a breaking change might as well do it in 2 places!

I'm leaning towards saying let's do it and take approach nr.1 - but @alethenorio what do you think?

Reply:
Yeah let's take a breaking change on this, I think that's OK given the size of the library and likely not a huge user base

Reply:
@parmus JUst mark the commit as breaking as well

Reply:
@odsod @alethenorio Actually, I just realized that there's a reason for this: this parameter can take multiple different formats: https://www.here.com/docs/bundle/routing-api-v8-api-reference/page/index.html#tag/Routing/operation/calculateRoutes

> Specifies the time of departure as defined by either date-time or full-date T partial-time in RFC 3339, section 5.6 (for example, 2019-06-24T01:23:45).
>
> The requested time is converted to local time at origin. When the optional timezone offset is not specified, time is assumed to be local. The special value any can be used to indicate that time should not be taken into account during routing. If neither departureTime or arrivalTime are specified, current time at departure place will be used. All time values in the response are returned in the timezone of each location.
>
> Note: Only long-term traffic incidents will be used if departureTime=any and traffic[mode]=default or no traffic[mode] are specified.

So there are actually three case:
1) Time specified
2) Time not specified => current time of departure will be used
3) `"any"` specified which changes what Here Maps takes into account.

Thoughts?

Reply:
We could add a new type, which handles this logic, but it's starting to feel a bit out-of-scope for this PR. I'm happy to make a second separate PR to try and address this.

Reply:
spontaneous thoughts: probably separate struct fields with mutual exclusivity validation, or only support one format

Reply:
I'm thinking that it might make sense to keep as a string for now and take some time to think through how we want to change for something more robust. WDYT @odsod

Reply:
Sounds good to me as well. @odsod, any final thoughts on this?

Reply:
Cool. Then I'm keeping it as a string for now, and then we can address this in a later PR.